lib/common.py

```diff
@@ -16,6 +17,15 @@ def vm_image(vm_key):
         url = DEF_VM_URL + url
     return url
 
+def prefix_object_name(label):
+    try:
+        from data import OBJECTS_NAME_PREFIX
+        name_prefix = OBJECTS_NAME_PREFIX
+    except ImportError:
+        name_prefix = f"[{getpass.getuser()}]"
+    object_name = f"{name_prefix} {label}"
```

Comment:
you don't need the `object_name` variable here as you don't use it for anything but returning it. You can return `f"{name_prefix} {label}"` directly.